components/chain.tsx

```diff
@@ -46,7 +46,7 @@ export default function Chain({ chain }) {
 
       <table>
         <tr>
-          <th>Height</th>
+          <th>Relative<br/>Height</th>
```

Comment:
Since we're not getting the full chain, I didn't find an easy way to get the actual height, so instead this is just the relative height from the chain tip.